.github/workflows/ci.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           admin_subscription_token: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_admin_token_by_role.subscription }}
           admin_support_token: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_admin_token_by_role.support }}
           admin_version_maintenance_token: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_admin_token_by_role.version-maintenance }}
+          list_tests: "[^(TestAccTFESAMLSettings_omnibus)]"
```

Comment:
I thought this was a pretty clever way skip, given that the issue with the test is that it cannot run concurrently.

I also think this will be hard to understand in the future without explanation.

A self-explaining method might be to set an environment variable when running go test and then checking that variable inside the test itself. Something like "CI_PUSH" or "CI_NIGHTLY" or something that could be used to "skipUnlessNightly"

Of course, you could also just document this option here inside the workflow, although I find the test name itself to be a very brittle form of test selection.

Reply:
`skipUnlessEnterprise()` is already our `skipUnlessNightly` so i don't understand what we're adding that's new here?

Reply:
I think using `skipIfEnterprise` is a little misleading because we are not skipping the tests because the feature is unavailable in Terraform Enterprise. Seems like we are solving for a different problem. Preventing these tests from running where we build our test setup seems clearer than having to maintain a custom solution, in my very humble opinion.

Reply:
RE: brittle name
I don't think we change tests names often but I can add a comment in the test file that notes we skip this test and should the test name ever change we need to update it in this file too?